stglib/mc.py

```diff
@@ -0,0 +1,188 @@
+import pandas as pd
+import xarray as xr
+
+from .core import qaqc, utils
+
+
+def read_asc(filnam, skiprows=50, encoding="utf-8"):
+    """Read data from an SBE 37 MicroCAT .asc file into an xarray
+    Dataset.
+
+    Parameters
+    ----------
+    filnam : string
+        The filename
+    skiprows : int, optional
+        How many header rows to skip. Default 50
+    encoding : string, optional
+        File encoding. Default 'utf-8'
+    Returns
+    -------
+    xarray.Dataset
+        An xarray Dataset of the MicroCAT data
+    """
+
+    df = pd.read_csv(
+        filnam,
+        skiprows=skiprows,
+        header=None,
+        names=["Temp", "Cond", "Sal", "Date", "Time"],
+        parse_dates={"time": ["Date", "Time"]},
+        encoding=encoding,
+        index_col=False,
+    )
+    print(df)
+    df.index.names = ["time"]
+    mc = xr.Dataset.from_dataframe(df)
+    return mc
+
+
+# Make raw CDF
```

Comment:
If you have a comment for the function, please put it as a docstring-style comment (delimited by three quote marks) within and immediately below the function definition. Similar to what you did with read_asc.